paperqa/agents/tools.py

```diff
@@ -373,6 +370,10 @@ async def complete(
         # TODO: eliminate race condition here if agent calls 2+ times in parallel
         # with opposite has_successful_answer values
         state.session.has_successful_answer = has_successful_answer
+
+        if not state.session.answer:
+            state.session.answer = CANNOT_ANSWER_PHRASE
```

Comment:
I don't know if "I cannot answer" makes sense here -- it's not that it could not, it's that it did not

Maybe make another sentinel `DID_NOT_ANSWER_PHRASE = "No answer given."`

Reply:
agreed -- I was trying to make this as simple as possible, but it's better to add some extra stuff.